### PR TITLE
Add scheduled workflow for preview

### PR DESCRIPTION
**Pull request by TravisEz13**
# PR Summary

### CODE REVIEW

File: .github/workflows/composite/createPR/action.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        with:
+          commit-message: "Update ${{ env.FORMULA_NAME}} formula to version ${{ env.NEW_FORMULA_VERSION }}"
+          token: "${{ secrets.PR_PAT }}"
+          committer: PwshBot <pwshbot@microsoft.com>
+          author: PwshBot <pwshbot@microsoft.com>
+          title: "Update ${{ env.FORMULA_NAME }} formula to version ${{ env.NEW_FORMULA_VERSION }}"
+          base: master
+          draft: false
+          branch: powershell-${{ env.CHANNEL_NAME }}-patch
+          push-to-fork: pwshBot/Homebrew-Tap
```
Comment (TravisEz13) [inline .github/workflows/composite/createPR/action.yml:24]:
```suggestion
          push-to-fork: pwshBot/Homebrew-Tap

```

File: .github/workflows/cron-powershell-preview.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  HOMEBREW_NO_ENV_HINTS: 1
+  HOMEBREW_NO_INSTALL_CLEANUP: 1
+
+jobs:
+  homebrew-formula-preview:
+    timeout-minutes: 15
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
[the remark this answers is not in the document]
Comment (adityapatwardhan) [reply .github/workflows/cron-powershell-preview.yml:37]: ?

File: .github/workflows/cron-powershell-preview.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  push:
+    paths:
+      - '.github/workflows/powershell-preview.yml'
+      - 'Formula/powershell-preview.rb'
+    branches:
+      - master
+  pull_request:
+    paths:
+      - '.github/workflows/powershell-preview.yml'
+      - 'Formula/powershell-preview.rb'
```
Comment (TravisEz13) [inline .github/workflows/cron-powershell-preview.yml:18]:
```suggestion
on:
  workflow_dispatch:
  schedule:
    # At 20:20 UTC on every day-of-week from Monday through Friday.
    - cron:  '0 0/4 * * *'
```

File: .github/workflows/cron-powershell-preview.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (TravisEz13) [reply .github/workflows/cron-powershell-preview.yml:31]: This is from a trusted publisher

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        ".github/workflows/**/*.yml": "Github Workflow"
+    }
+}
```
Comment (TravisEz13) [inline .vscode/settings.json:5]:
```suggestion
}

```

**Comment by JamesWTruher** [comment]
this looks good!

**Review by JamesWTruher — APPROVED** [review]

**Pull request merged**